Sakura/core/payments.py

```
@@ -0,0 +1,116 @@
+                telegram_payment_charge_id=payment.telegram_payment_charge_id
+            )
+            keyboard = [[InlineKeyboardButton("Buy flowers again 🌸", callback_data="get_flowers_again")]]
+            reply_markup = InlineKeyboardMarkup(keyboard)
+            refund_msg = random.choice(REFUND_MESSAGES)
+            if update.message.chat.type == "private":
+                try:
+                    url = f"https://api.telegram.org/bot{BOT_TOKEN}/sendMessage"
+                    payload = {
+                        'chat_id': update.message.chat.id,
+                        'text': refund_msg,
+                        'message_effect_id': random.choice(EFFECTS),
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)